packages/machine/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import { InstructionExecutor } from "./instruction-executor";
 
 export { InstructionExecutor };
+export * from "./models";
```

Comment:
We need to finish reviewing the patterns used in the Machine's models before we expose them as public API